File: Go/ContaBancaria.go
package listaRicardo

import "fmt"

type ContaBancaria struct {
	Saldo   float32
	Titular string
}

func (c *ContaBancaria) SetSaldo(saldo float32) {
	c.Saldo = saldo
}

func (c *ContaBancaria) GetSaldo() float32 {
	return c.Saldo
}

func (c *ContaBancaria) SetTitular(titular string) {
	c.Titular = titular
}

func (c *ContaBancaria) GetTitular() string {
	return c.Titular
}

func (c *ContaBancaria) Depositar(valor float32) {
	c.Saldo += valor
}

func (c *ContaBancaria) Sacar(valor float32) {
	c.Saldo -= valor
}

func (c *ContaBancaria) MostrarSaldo() {
	fmt.Printf("O saldo atual é de: %.2f\n", c.Saldo)
}

func main() {
	conta := ContaBancaria{Saldo: 1000.0, Titular: "João"}

	conta.Depositar(500)
	conta.MostrarSaldo()

	conta.Sacar(200)
	conta.MostrarSaldo()
}

File: Java/ContaBancaria.java
public class ContaBancaria {
    private float saldo;
    private String titular;

    public ContaBancaria(float saldo, String titular){
        this.saldo = saldo;
        this.titular = titular;
    }

    public void  setSaldo(float saldo){
        this.saldo = saldo;
    }

    public float getSaldo() {
        return saldo;
    }

    public void  setTitular(String titular){
        this.titular = titular;
    }

    public String getTitular() {
        return titular;
    }

    public void depositar(int adicionar){
        this.saldo += adicionar;
    }

    public void sacar(int retirar){
        this.saldo += retirar;
    }

    public void mostrarSaldo(){
        System.out.println("O saldo atual é de: "+ this.saldo);
    }



}
